tables/cuneiform-transliteration-academic.utb

```diff
@@ -0,0 +1,713 @@
+# Cuneiform Transliteration and Normalization (Academic)
+#
+#-index-name: Cuneiform (Academic)
+#-display-name: Cuneiform Transliteration and Normalization,  Academic
+#
+#+language: akk-Latn
+#+language: xhu-Latn
+#+language: uga-Latn
+#+language: peo-Latn
+#+language: sux-Latn
+#+language: xeb-Latn
+#+language: elx-Latn
+#+language: hit-Latn
+#+language: xur-Latn
+#+language: xlu-Latn
```

Comment:
I want to make sure that this table actually target all of these languages. In addition, are there any languages missing from this list? In the description below you mention Hebrew, Aramaic, Coptic, ... I don't see these here.